jaguar2-examples/pom.xml

```
@@ -32,6 +32,9 @@
 		<module>jaguar2-example-junit4</module>
 		<module>jaguar2-example-junit4-ba-dua</module>
 		<module>jaguar2-example-junit4-jacoco</module>
+		<module>jaguar2-example-cli-junit4</module>
+		<module>jaguar2-example-cli-junit4-ba-dua</module>
+		<module>jaguar2-example-cli-junit4-jacoco</module>
 	</modules>
 
```

```suggestion
		<module>jaguar2-example-cli-junit4</module>
		<module>jaguar2-example-cli-junit4-jacoco</module>
	</modules>

	<profiles>
		<profile>
			<id>ba-dua-cli-examples</id>
			<activation>
				<jdk>[,22)</jdk>
			</activation>
			<modules>
				<module>jaguar2-example-cli-junit4-ba-dua</module>
			</modules>
		</profile>
	</profiles>

```

Let's make `jaguar2-example-cli-junit4-ba-dua` only run for Java < 22 until we fix https://github.com/saeg/ba-dua/issues/41